frontend/src/components/Pages/RobotPage/LocalizationDialog.tsx

```diff
@@ -97,11 +97,10 @@ export const LocalizationDialog = ({ robot }: RobotProps): JSX.Element => {
 
     const onClickLocalize = () => {
         if (selectedAssetDeck && localisationPose) {
-            BackendAPICaller.postLocalizationMission(localisationPose, robot.id)
+            BackendAPICaller.postLocalizationMission(localisationPose, robot.id, selectedAssetDeck.id)
```

Comment:
Make a corresponding structure in frontend as in backed for ScheduleLocalizationMissionQuery

Reply:
Not sure if its required for this to work. Maybe it is a better approach will look into it.

Reply:
Would make more sense if the value was fetched from backend and sent to frontend unlike this case where it is the other way around. Dont you agree. ? :p